.gitignore

```diff
@@ -9,4 +9,5 @@ tmp
 vendor/
 .vscode/
 .idea/
-.env
\ No newline at end of file
+.env
+test/scratch_test.go
```

Comment:
suggestion: let's check that test in, but `t.Skip("only run locally")` it right away.
Maybe read the hosts + API keys from environment so as not to expose them.

Then the test could be re-run another time without rewriting it.